### PR TITLE
Bugfixing and new feature

### PR DESCRIPTION
**Pull request by Xento**
- Fixed bug when receiving Off Button when an other group was selected
  before.
- Don't set color and brightness when turning on via zone 0
- reflect all changes made to zone 0 to all other zones

### CODE REVIEW

File: devices/milightRF24.coffee [hunk 1]
```diff
@@ -78,7 +144,26 @@ module.exports = (env) ->
           button = Buttons.Group4Off
       
       @gateway.sendButton(id, zone, button, false)
-    
+      
+      dataObj = {
```
Comment (philip1986) [inline devices/milightRF24.coffee]: Too much code duplication! I would suggest a method like: 

```
_getDataObj: (id, zone, button, brightness = 0, r = 0, g = 0, b = 0) ->
       raw: 'loop'
        id: id,
        zone: zone
        button: button
        longPress: fals,
        discoMode: 0
        brightness: brightness,
        color:
          r:  r
          g: g
          b: b

```

Comment (Xento) [inline devices/milightRF24.coffee]: I will fix it after my holiday or you can do it if you like ;-)

Comment (Xento) [inline devices/milightRF24.coffee]: Fixed it now, but there is still a bug, that the white mode is not set on all other zones when you set it in zone 0

**Comment by Xento** [comment]
I think you can now merge my code ;-)

**Comment by philip1986** [comment]
Yes, it looks better now. Thx for changing.

**Pull request closed without merging**